### DISCUSSION TITLE
When trying to flash BIOS, it says not a proper BIOS

### DISCUSSION THREAD

**Discussion by MrAnonymous7122** [xCuri0/ReBarUEFI#166] [Q&A]
**System**
 - Motherboard: H110M-R
 - BIOS Version: 4210 I think (it's the latest version)
 - GPU: Sapphire RX 580 4GB
 - [Y] CSM is turned off. **Make sure to confirm this in the BIOS and not with GPU-Z or similar since it can be inaccurate**
 - [N] 4G decoding is enabled. **Make sure to confirm this in the BIOS and not with GPU-Z or similar since it can be inaccurate**
 - [Y] UEFIPatch is applied (see [Using UEFIPatch](https://github.com/xCuri0/ReBarUEFI/wiki/Using-UEFIPatch) for more information). On some motherboards [DSDT Patching](https://github.com/xCuri0/ReBarUEFI/wiki/DSDT-Patching) is also needed
 - [Y] I have read [Common issues (and fixes)](https://github.com/xCuri0/ReBarUEFI/wiki/Common-issues-(and-fixes))

**Description**

Describe the issue here.
I followed the instructions to add the ffs module and patch it and end up with a BIOS image.
My BIOS did not accept .patched as a file format for image(it only accepts .cap)
I did patch first and then add ffs but it says this image is not a proper image.
Previously when using UEFITool, it produced a .rom file which my BIOS did not accept.
To make it a .cap file, I set the name to etc.CAP when saving it after adding module.
There may be something simple I am missing but any help is appreciated.

**Comment by histfuture** [comment]
same question

**Comment by gervisc** [comment]
see [https://winraid.level1techs.com/t/guide-how-to-flash-a-modded-ami-uefi-bios/30627] 
in case of asus I used usb flashback and https://biosrenamer.stouder.io/
